### DISCUSSION TITLE
Some folders are not synced between host and container

### DISCUSSION THREAD

**Discussion by iranimij** [wardenenv/warden#796] [Help & Support]
Some folders like var and pub/assets are not synced between host and container,
For example if I create a folder in the root it also creates it inside host, but when for example I create var folder it does not appear in host.

I am using Mac sonoma 14.6

**Comment by navarr** [comment]
@bap14 did a good write up on how to do this, and it's clear I need to add it to the documentation - but this is expected behavior to ensure mutagen speed.

HOWEVER, you can overwrite this behavior by adding a custom Mutagen file.

> you can create a `.warden/mutagen.yml` file in your project and it will be used instead of the default Magento environment mutagen configuration file. You can start by copying the [mutagen file from the Magento 2 template](https://github.com/wardenenv/warden/blob/main/environments/magento2/magento2.mutagen.yml) or just start fresh. Then just restart your environment and it should pick up your new configuration.

OP: https://github.com/orgs/wardenenv/discussions/782#discussioncomment-10001868

**Comment by iranimij** [comment]
Thanks, It would be great if you could add it to the documentation.
